docs/index.rst

```diff
@@ -114,6 +114,14 @@
 .. toctree::
   :maxdepth: 0
   :hidden:
+  :caption: Stackdriver Error Reporting
```

Comment:
To clarify, just "Monitoring" is fine, so I could actually just call this "Error Reporting" as well. Just not Cloud Monitoring/Cloud Error Reporting. I'll just change this to Error Reporting for now.